consensus/src/core.rs

```diff
@@ -300,9 +300,7 @@ impl Core {
 
         // Check if we can commit the head of the 2-chain.
         // Note that we commit blocks only if we have all its ancestors.
-        let mut commit_rule = b0.round + 1 == b1.round;
-        commit_rule &= b1.round + 1 == block.round;
-        if commit_rule {
+        if b0.round + 1 == b1.round {
```

Comment:
Are the QCs of b0 and b1 already verified?

Reply:
If by "verified" you mean that all signatures check, then yes.

Reply:
Is that what you meant?